### PR TITLE
grant raphael.gomes@octobus.net access to Mercurial

### PR DESCRIPTION
**Pull request by martinvonz**

### CODE REVIEW

File: projects/mercurial/project.yaml [hunk 1]
```diff
@@ -6,9 +6,10 @@ auto_ccs:
...
   - "martinvonz@google.com"
+  - "raphael.gomes@octobus.net"
 sanitizers:
   - address
   - undefined
 # Disabled MSAN because of https://github.com/google/oss-fuzz/issues/6294
 #  - memory
-coverage_extra_args: -ignore-filename-regex=.*/sanpy/.*
\ No newline at end of file
+coverage_extra_args: -ignore-filename-regex=.*/sanpy/.*
```
Comment (martinvonz) [inline projects/mercurial/project.yaml:15]: GitHub's editor added the newline. Seems like a good thing anyway.

**Comment by DavidKorczynski** [comment]
The CI fails because there is no `main_repo` field in `project.yaml` - could you add this too?

**Comment by martinvonz** [comment]
A presubmit check failed because the Mercurial project is missing the `main_repo` field. I've attempted to fixed that in a separate PR: https://github.com/google/oss-fuzz/pull/6510. Let's see if that works or if it fails because it's not a Git repo.

**Comment by martinvonz** [comment]
> The CI fails because there is no `main_repo` field in `project.yaml` - could you add this too?

Sorry, didn't see this until now, but as you already know, it was fixed in #6510.

**Review by DavidKorczynski — APPROVED** [review]

**Comment by DavidKorczynski** [comment]
> > The CI fails because there is no `main_repo` field in `project.yaml` - could you add this too?
> 
> Sorry, didn't see this until now, but as you already know, it was fixed in #6510.

No problem - thanks for the merge! LGTM

**Comment by Alphare** [comment]
Hi, thanks for being so quick. It appears I have an issue. I initially registered my Google account with an alias (rgomes@octobus.net), I've just changed it to raphael.gomes@octobus.net and oss-fuzz still says I'm rgomes@octobus.net. Is there any way I can change that association? Thanks

**Comment by martinvonz** [comment]
> Hi, thanks for being so quick. It appears I have an issue. I initially registered my Google account with an alias ([rgomes@octobus.net](mailto:rgomes@octobus.net)), I've just changed it to [raphael.gomes@octobus.net](mailto:raphael.gomes@octobus.net) and oss-fuzz still says I'm [rgomes@octobus.net](mailto:rgomes@octobus.net). Is there any way I can change that association? Thanks

I have very little experience with oss-fuzz myself. What do you mean by "oss-fuzz still says I'm rgomes@octobus.net"? Where does it say that? It's not just that you need to add/change the new email address in GitHub's account settings?

**Comment by DavidKorczynski** [comment]
So the emails in the `project.yaml` need to be affiliated with a Google account and the email you use in `project.yaml` should be the primary email of that account. Please see here: https://google.github.io/oss-fuzz/getting-started/new-project-guide/#primary - is `raphael.gomes@octobus.net` your primary email on that account? 

See here as well: https://google.github.io/oss-fuzz/faq/#why-do-you-require-a-google-account-for-authentication and in particular "*Note that an alternate email address associated with a Google account does not work due to appengine api limitations.* "

**Comment by Alphare** [comment]
I had to switch account from Monorail in order for it to pick up the primary address change. I can access the report now, thanks!

**Comment by Alphare** [comment]
Scratch that, it only works for bugs.chromium.org, not oss-fuzz.com. Sorry for the noise. raphael.gomes@octobus.net is indeed my primary email on my account, but it was rgomes@octobus.net when I first tried to login, it seems that the login on oss-fuzz is not picking this change up.

**Pull request merged**